Resources/Prototypes/DeltaV/Voice/speech_emotes.yml

```diff
@@ -2,242 +2,78 @@
 - type: emote
   id: HarpyHonk
   category: Vocal
-  chatMessages: [honks.]
+  chatMessages: [ honks ]
   chatTriggers:
-    - honk
-    - honk.
```

Comment:
Slightly overzealous here. The singular form honk and past-tense honked are still necessary.

Reply:
why?